debian/ceph-base.postinst

```diff
@@ -33,13 +33,15 @@ case "$1" in
 	rm -f /etc/init/ceph.conf
 	[ -x /sbin/start ] && start ceph-all || :
 
-        # adjust file and directory permissions
-	for DIR in /var/lib/ceph/* ; do
```

Comment:
Why not just add the missing `/var/lib/ceph/crash` folder here? The new code does treat this folder individually.

Reply:
IIUC, what we need to do is to fix the permissions of the directories listed by `debian/ceph-base.dirs`, right? if that's the case, i concur with @frittentheke , that probably we can just add `var/lib/ceph/crash` here, like
```sh
for DIR in /var/lib/ceph/* /var/lib/ceph/crash/posted; do
    if ! dpkg-statoverride --list $DIR >/dev/null
    then
        chown $SERVER_USER:$SERVER_GROUP $DIR
    fi
done
```

please note, i am listing `/var/lib/ceph/crash/posted` here, as `/var/lib/ceph/crash` has been covered by `/var/lib/ceph/*`.

Reply:
> Why not just add the missing `/var/lib/ceph/crash` folder here? The new code treats this folder individually.

That's actually a very fair point. In some iteration of this patch I ended up using the `find`+`xargs` approach, as that seemed just more logical at the time. (IIRC it was a globbing thing first that then became redundant anyway; the `find`+`xargs` combo remained anyway.)


> ```shell
> for DIR in /var/lib/ceph/* /var/lib/ceph/crash/posted; do
>     if ! dpkg-statoverride --list $DIR >/dev/null
>     then
>         chown $SERVER_USER:$SERVER_GROUP $DIR
>     fi
> done
> ```
> 
> please note, i am listing `/var/lib/ceph/crash/posted` here, as `/var/lib/ceph/crash` has been covered by `/var/lib/ceph/*`.

While this does obviously `chown` the `crash` dir, it doesn't update the already existing entries in `crash/posted` that might've been created as `root` prior to #48713. That's why I chose to treat `/var/lib/ceph/crash/` separately.